README.md

```diff
@@ -3,3 +3,38 @@
 
 
 Django model and view to store, generate and serve configuration for MapBox GL JS map base layer
+
+
+# Getting started
+### Installation
+Install it in your project like any dependency
+```
+pip install django-mapbox-baselayer
+```
+
+### Usage
+Declare django-mapbox-baselayer in the `INSTALLED_APPS`
+```py
+# settings.py
+
+INSTALLED_APPS  = [
+    # ... other django apps
+    "mapbox_baselayer",
+]
+```
+
+/!\ Note: for now, the apps is not registered in the admin. But you can register it in your project if you want to use it
+```py
+# admin.py
+
+from django.contrib import admin
+from mapbox_baselayer.admin import MapBaseLayerAdmin
+from mapbox_baselayer.models import MapBaseLayer, BaseLayerTile
+
+admin.site.register(MapBaseLayer, MapBaseLayerAdmin)
+admin.site.register(BaseLayerTile)
```

Comment:
useless. Tile can already be defined in MapBaseLayerAdmin, as Inline data

Reply:
I removed it.